src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -39,13 +40,32 @@
     ("dateTime", "long")
 ]
 
+# Data used in fill_missing_satellite_subidentifier from file
+# names that use satDAAC identifiers, like the UCAR/COSMIC archive.
+# The first parentheses in the pattern should match the satId
+# and the second parentheses should match the number (subidentifier).
+# Satellite subidentifiers are typically not needed for government
+# missions. For example, Cosmic1 (C[0-9]{3}) and Cosmic2 (C2E[0-9])
+# do not use subidentifiers.
+satIdPatterns = [
+    r'^(GN)([0-9]{2})$',     # PlanetiQ
```

Comment:
```suggestion
    r'^(GN|YM)([0-9]{2})$',     # PlanetiQ
```

and here too @steve-marshall-piq

Reply:
@BenjaminRuston - This looks good.  Both GN and YM are prefixes used for different PlanetiQ satellites.